README.md

```diff
@@ -1,20 +1,45 @@
+The xDS vision is one of a universal data plane API, articulated at
+[https://blog.envoyproxy.io/the-universal-data-plane-api-d15cec7a](https://blog.envoyproxy.io/the-universal-data-plane-api-d15cec7a).
+xDS aims to provide a set of APIs that provide the de facto standard for L4/L7
+data plane configuration, similar to the role played by OpenFlow at L2/L3/L4 in
+SDN.
 
-The APIs will be defined canonically in proto3 and incrementally evolve from [existing Envoy xDS APIs](https://github.com/envoyproxy/data-plane-api) via a well defined [stable API versioning policy](https://docs.google.com/document/d/1xeVvJ6KjFBkNjVspPbY_PwEDHC7XPi0J5p1SqUXcCl8/edit#). APIs will cover service discovery, load balancing assignments, routing discovery, listener configuration, secret discovery, load reporting, health check delegation, etc.
+The [existing Envoy xDS
+APIs](https://github.com/envoyproxy/envoy/tree/master/api)
+constitute the basis for this vision and will incrementally evolve towards
+supporting a goal of client neutrality. The xDS APIs have two components, a
+transport protocol and data model. The data model covers common data plane
+concerns such as service discovery, load balancing assignments, routing
+discovery, listener configuration, secret discovery, load reporting, health
+check delegation, etc.
 
-We will evolve and mold the APIs to support client-side lookaside load balancing (e.g. gRPC-LB), data plane proxies beyond Envoy, hardware LB, mobile clients and beyond. We will strive to be vendor and implementation agnostic to the degree possible while not regressing on support for projects that have committed to UDPA in production (Envoy & gRPC-LB so far).
+We will evolve the xDS APIs to support additional clients, for example data
+plane proxies beyond Envoy, proxyless service mesh libraries, hardware load
+balancers, mobile clients and beyond. We will strive to be vendor and
+implementation agnostic to the degree possible while not regressing on support
+for data plane components that have committed to xDS in production (Envoy & gRPC to date).
 
-# Membership
+# Repository structure
 
-Initial members will include representatives from the Envoy and gRPC projects. This will include Google and Lyft based maintainers, as well as members from Microsoft and Amazon. We are soliciting the wider data plane proxy community for additional interest in initial membership, since we feel that a truly universal API should reflect a diverse set of projects, organizations and individuals.
+The xDS APIs are split between this repository and
+https://github.com/envoyproxy/envoy/tree/master/api. We aim to opportunistically
```

Comment:
I think we should say something a bit stronger here.  For example, we should explicitly state that our long-term goal is to move the entire API to this repo, but that we are doing so opportunistically over time as we generalize parts of the API to be less Envoy-specific.